lib/services/services_lsb.c

```diff
@@ -48,8 +48,6 @@
                            " " PCMK_META_TIMEOUT "='15s' />\n"                \
     "    <" PCMK_XE_ACTION " " PCMK_XA_NAME "='restart'"                      \
                            " " PCMK_META_TIMEOUT "='15s' />\n"                \
-    "    <" PCMK_XE_ACTION " " PCMK_XA_NAME "='force-reload'"                 \
-                           " " PCMK_META_TIMEOUT "='15s' />\n"                \
```

Comment:
The force-reload timeout doesn't come from the agent (the LSB script) though, it's just something Pacemaker is making up. It's hard for me to imagine anything wanting to use it since it's not true information, or why something would want to pull LSB metadata from Pacemaker OCF metadata in general. (The other direction is reasonable but Pacemaker does that.)

No problem to add this back though...

Reply:
standing by my statement but adding this back for now to get through review

Reply:
The `<actions>` section advertises what actions the agent supports, so we're listing everything required by the LSB standard (even though init scripts often aren't fully compliant in practice, especially since the only platforms still using them at boot don't follow LSB). OCF metadata requires name and timeout for advertised actions, but LSB doesn't provide timeouts, so we have to make up something. The LSB-specific metadata is unnecessary but I guess since they had to parse the header they decided to grab all of it. All of that is of questionable value but it makes some sense :)